mmv1/third_party/terraform/website/docs/r/service_networking_connection.html.markdown

```diff
@@ -27,11 +27,20 @@ resource "google_compute_global_address" "private_ip_alloc" {
   network       = google_compute_network.peering_network.id
 }
 
-resource "google_service_networking_connection" "foobar" {
+resource "google_service_networking_connection" "default" {
   network                 = google_compute_network.peering_network.id
   service                 = "servicenetworking.googleapis.com"
   reserved_peering_ranges = [google_compute_global_address.private_ip_alloc.name]
 }
+
+# Import or export custom routes
+resource "google_compute_network_peering_routes_config" "peering_routes" {
+  peering = google_service_networking_connection.default.peering
+  network = google_compute_network.peering_network.name
+
+  import_custom_routes = true
+  export_custom_routes = true
+}
```

Comment:
Yeah, I agree this may be useful for users but this sounds more like an example for `google_compute_network_peering_routes_config` resource and we've already have this use case covered in https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network_peering_routes_config#example-usage---network-peering-routes-config-basic. Will that be enough for users?
I think the example usage in this page is designed to be a basic configuration of `google_service_networking_connection` (as it may also be used in many other cases). Please correct me if that's not the case.

Reply:
The official doc has a section that mentions importing and exporting routes but doesn't show how. I think this is a very helpful improvement on the sample. I added "Optional" to the comment.

https://cloud.google.com/vpc/docs/configure-private-services-access#on-prem